TODO/6-practical-skills-for-ux-designers.md

```diff
@@ -1,85 +1,88 @@
 > * 原文地址：[6 Practical Skills for UX Designers](https://uxdesign.cc/6-practical-skills-for-ux-designers-22c852d6c576#.vjeb02dwq)
 * 原文作者：[Joanna Ngai](https://uxdesign.cc/@ngai.yt)
 * 译文出自：[掘金翻译计划](https://github.com/xitu/gold-miner)
-* 译者： 
+* 译者： [Kulbear](https://kulbear.github.io/)
 * 校对者： 
 
 ![](http://ac-Myg6wSTV.clouddn.com/2cc6a114bae9326ef2b0.png)
 
-### 6 Practical Skills for UX Designers
+### 用户体验设计师的 6 个实用技能
 
 #### Thoughts on how to transform the way you develop products, services and processes
 
 ![](http://ac-Myg6wSTV.clouddn.com/66ff18b264d2c507aebe.png)
 
-While we can all agree that the way [senior designers and junior designers approach problems is markedly different](https://medium.com/the-year-of-the-looking-glass/junior-designers-vs-senior-designers-fbe483d3b51e#.4a2tc78vd), what exactly is it that separates the novices from the seasoned pros?
+我们都同意[中高级设计师和初级设计师处理问题的方法有着显著区别](https://medium.com/the-year-of-the-looking-glass/junior-designers-vs-senior-designers-fbe483d3b51e#.4a2tc78vd)，到底是什么东西使我们能区分开新人和久经沙场的老设计师们呢？
 
-Here are a few practical skills to hone as you embark on your journey as a designer.
+接下来是一些在你从事设计师的这条漫长旅途上需要磨砺的一些实用技能。
 
-### 1. View design solutions from 10,000 feet
+### 1. 从长远的角度看你的设计方案
 
-As I mentioned [previously](https://blog.prototypr.io/essential-lessons-on-ux-18f96933e885#.mjgjp0osb), designers need to have perspective and to understand complex problems, without getting bogged down by the details too early on.
+如我[之前](https://blog.prototypr.io/essential-lessons-on-ux-18f96933e885#.mjgjp0osb)所提及的，设计师们需要在理解复杂的问题，拥有自己的看法的同时让自己免于陷入考虑多过细节的泥潭。
 
-> A mentor of mine remarked that designers do more than just motivate a user to click a button or even complete a task. UX designers must view the design problem and have a frame of mind **“10,000 feet” above**.
+> 我的一位导师提过，设计师们做的不仅仅是引导用户去点击一个按钮或是完成一个小任务。用户体验设计师必须从长远角度看待设计问题，并拥有一个**“提前 10000 英尺” 的框架**。
 
-> It considers previous thoughts, attitudes, competitors and other tools that operate within the system that the user interacts with.
+> 也就是需要在用户交互的这个系统中考虑以前的想法，态度，竞争对手和其他工具。
 
-Consider context rather than your specific scenario. void designing in a silo no matter what stage of the design progress you are in.
+考虑全局（上下文？）而非眼下的这个特殊情景。void designing in a silo（这句不会。。。。） 无论你处在设计的哪个阶段。
```

Comment:
> Consider context rather than your specific scenario. void designing in a silo no matter what stage of the design progress you are in.

建议翻译成

> 在你设计的时候，要从全局考虑用户所处的环境，而不是考虑某个特定的情况；不然，不论你处在设计的那个阶段，都应该把你现在的设计丢进垃圾桶里。